src/Microsoft.TestPlatform.CrossPlatEngine/Client/ProxyExecutionManager.cs

```diff
@@ -152,13 +152,19 @@ public virtual int StartTestRun(TestRunCriteria testRunCriteria, ITestRunEventsH
             catch (Exception exception)
             {
                 EqtTrace.Error("ProxyExecutionManager.StartTestRun: Failed to start test run: {0}", exception);
+
+                // Log error message to design mode and CLI.
+                var testMessagePayload = new TestMessagePayload { MessageLevel = TestMessageLevel.Error, Message = exception.ToString() };
+                this.HandleRawMessage(this.dataSerializer.SerializePayload(MessageType.TestMessage, testMessagePayload));
```

Comment:
nit: I feel that the commandline does not need to pay this serialization cost since HandleRawMessage in that case would be a no-op. However since this is a one time thing I think it should be ok.